src/test/scenarios/managed-network/configuration/readme.az.md

```diff
@@ -60,4 +35,16 @@ directive:
       $['x-ms-client-flatten'] = true;
     reason: Flatten everything for Azure CLI
 
+clicommon:
+    cli-directive:
+    # directive on operationGroup
+      - select: 'operationGroup'
+        where:
+            operationGroup: 'operations'
+        hidden: true
+      - where:
+            operationGroup: 'managed_networks'
+            operation: 'list_by_resource_group'
```

Comment:
why this is removed?

Reply:
testing. there's no other command with more than one method.